submissions/vovan-zt/TiniWorld2/index.js

```diff
@@ -0,0 +1,84 @@
+class WorldInhabitant {
+	constructor(species, name, gender, saying) {
+		this.species = species;
+		this.name = name;
+		this.gender = gender;
+		this.saying = saying;
+	}
+
+	getMeaning() {
+		return ["species", "name", "gender", "saying"]
+			.map((property) => {
+				if (property === "species") {
+					return this[property].toUpperCase();
+				}
+				return this[property];
+			})
+			.join("; ");
```

Comment:
```
[
  "Some property: ",
  this.property,
  "Uppecased property: ",
  this.property.toUpperCase(),
  "A list of something: ",
  Array.isArray(this.property) ? this.property.join(",") : "no data",
].join("glue-string");
```

Reply:
Can I do it like this?

getMeaning() {
		return [this.species.toUpperCase(), this.name, this.gender, this.saying].join("; ");
	}

Reply:
This will read well (provided also properly formatted).